app/build.gradle

```diff
@@ -97,6 +97,10 @@ android {
         androidTest.assets.srcDirs += files("$projectDir/schemas".toString())
     }
 
+    androidResources {
+        generateLocaleConfig = true
+    }
+
```

Comment:
If we don't pay attention to new languages being added from Weblate, this might lead to half-translated being proposed to the user. Currently the list of languages is manually kept in `settings_keys.xml`, so when new languages are added through Weblate they don't automatically appear in the app menu. However, I'm fine with switching to the new behavior, as it would avoid work to keep the language list up to date. Is it possible to access the locale config from within the app, so we can delete the list of languages in `settings_keys.xml` and use the autogenerated locale config even for the pre-Android-13 language picker?

Reply:
I found [LocaleManager#getOverrideLocaleConfig](https://developer.android.com/reference/android/app/LocaleManager#getOverrideLocaleConfig()) but it sounds like that method is only to get a `LocaleConfig` that's already been set with `setOverrideLocaleConfig()`. As far as I can tell, we would have to manually generate a `LocaleConfig` instead of using the auto-generated one if we wanted to use it for this.

Reply:
I could find the data under `app/build/generated/res/localeConfig/debug/xml/_generated_res_locale_config.xml` after running the build, with this data inside:
```xml
<locale-config xmlns:android="http://schemas.android.com/apk/res/android">
    <locale android:name="en-US"/>
    <locale android:name="ace"/>
    ...
```
So in theory the data is there, but in practice I think we better not access it by manually opening the xml since it's not officially supported. So yeah let's keep the previous behavior.